README.md

```diff
@@ -29,31 +27,96 @@ The `app_name` variable will be used to set the database names and credentials.
 
 You can also change the war used to deploy geoserver with the following variable.
 
-* `geoserver_url` - GeoServer war URL (default: `http://build.geonode.org/geoserver/latest/geoserver.war`)
+* `gs_war_url` - GeoServer war URL (default: `http://build.geonode.org/geoserver/latest/geoserver.war`)
 
-Dependencies
-------------
+You can tune PostgreSQL, Tomcat8, and NGINX with the following variables:
 
+* `pg_max_connections` - PostgreSQL Max Connections (default: `100`)
+* `pg_shared_buffers` - PostgreSQL Shared Buffers (default: `128MB`)
+* `tomcat_xms` - Tomcat JAVA_OPTS xms (default: `1024M`)
+* `tomcat_xmx` - Tomcat JAVA_OPTS xmx (default: `2048M`)
+* `nginx_client_max_body_size` - NGINX Client Max Body Size (default: `400M`)
 
-Example Playbook
-----------------
+The following security variables should be added to `ansible-playbook ...` as command line flags or stored securely outside of `ansible-geonode`, `geonode-project`, or your project repo.
 
-The following is an example playbook using variables. This playbook will be included in your geonode template project clone.
+* `dj_superuser_password` - Django Admin Password (default: `admin`)
+* `gs_admin_password` - GeoServer Admin Password (default: `geoserver`)
+* `gs_root_password` - GeoServer Root Password (default: `M(cqp{V1`)
+
+## Dependencies
 
-    - hosts: webservers
-        remote_user: ubuntu
-        vars:
-            app_name: my_geonode
-            github_user: GeoNode
-        roles:
-            - { role: GeoNode.geonode }
 
-License
--------
+## Example Playbook
```

Comment:
We can cut down the `playbook.yml` example as you like, but the pre_tasks are necessary and other people shouldn't have to spend hours hunting those down themselves.  I'd like it to work out of the box.

We could move the tomcat, postgres, etc. config to a `Production.md` file for showing and explaining what a production playbook looks like.